src/Microsoft.ML.Legacy/Runtime/EntryPoints/JsonUtils/ExecuteGraphCommand.cs

```diff
@@ -145,7 +145,11 @@ public void GetOutputToPath(GraphRunner runner, string varName, string path, Tlc
                     throw _host.ExceptNotSupp("File handle outputs not yet supported.");
                 case TlcModule.DataKind.DataView:
                     var idv = runner.GetOutput<IDataView>(varName);
-                    SaveDataView(idv, path, extension);
+                    if (idv != null)
+                        SaveDataView(idv, path, extension);
+                    else
+                        using (var ch = _host.Start("Get outputs from executed graph"))
+                            ch.Warning("Ignore empty graph output");
```

Comment:
>Ignore empty graph output [](start = 40, length = 25)

this is too broad.
Can you use path, extension, or var name to specify what exactly we ignoring?

Reply:
No problem!

---
In reply to: [240802449](https://github.com/dotnet/machinelearning/pull/1862#discussion_r240802449) [](ancestors = 240802449)